## Genome analysis


| Date       | Week | Process                               | Software                 | Estimated time    | 
| ---------- | ---- | ------------------------------------- | ------------------------ | ----------------- | 
| 04-04-2019 | 14   | Familiarize with files and UPPMAX     | --                       | 1 h               |
| 04-04-2019 | 14   | Quality check of reads                | FastQC                   | 3 min             |
| 05-04-2019 | 14   | Trimming of genomic and RNA reads     | Trimmomatic              | 1-10 min per file |
| 11-04-2019 | 15   | Assembly quality assessment           | BCFtools                 | 1.5 h             |
| 12-04-2019 | 15   | Genome annotation                     | Maker2                   | 3.5 h             |
| 04-04-2019 | 14   | Homology search                       | blastn                   | 1 h               |
| 04-04-2019 | 14   | Transcriptome assembly                | Trinity                  | 1.5 h             |
| 04-04-2019 | 14   | RNA mapping                           | TopHat                   | 5-30 min per file |            
| 04-04-2019 | 14   | Analyses of long non-coding RNAs      | See below                | --                |
| 04-04-2019 | 14   | Compare lncRNAs to databases          | lncRNAdb, GENCODE lncRNA | 1 h               |
| 04-04-2019 | 14   | Compare homogenic lncRNAs             | BLAST                    | 1 h               |
| 04-04-2019 | 14   | Calculate read counts                 | Htseq                    | 5 min             |
| 04-04-2019 | 14   | Test of differential expression       | Deseq2                   | Variable          |
| 04-04-2019 | 14   | Calculate heterozygosity of genome    | BWA                      | 1.5 h             |
| 04-04-2019 | 14   | Improve annotation with RNA assemblies| Maker2                   | 0.5-1.5 h         |
